README.md

```diff
@@ -33,6 +33,15 @@ The plugin will automatically use any of the following configuration variables,
 * `description` - A longer description of what your site is about, e.g., "Where I blog about Jekyll and other awesome things"
 * `url` - The URL to your site, e.g., `https://example.com`. If none is provided, the plugin will try to use `site.github.url`.
 * `author` - Global author information (see below)
+* `blog.icon` - Icon with 1:1 proportions for readers to use for the blog (not supported by all readers; often overridden by a favicon or site icon)
+* `blog.logo` - Logo with 2:1 proportions for readers to use for the blog (not supported by all readers). 
+  For example, both of the preceding can be expressed:
+  ```yml
+  blog:
```

Comment:
Hm, I think these should be under the `feed` key, not a `blog` key.

Reply:
No objection really, maybe a blog key with “feed-icon” and “feed-logo” sub-keys? I like the idea of keeping the blog info in one place, rather than having multiple keys.

```yml
blog:
  title: My awesome Blog
  feed-icon: /images/icon.png
  feed-logo: /images/logo.png
```

Reply:
This plugin uses some global keys like `title` but in general it keeps its own configuration in the `feed` key. So `site.feed.icon` in Liquid. Is there a commonly-used icon key we could use across multiple plugins? If this is only for a feed and would never be reused, then it should be under `feed`. No other Jekyll plugins use the `blog` key, IIRC.

Reply:
I’m not at all familiar with what other plugins do, so I would defer to you. For mY part, my blog has its own title, separate from my site, so I use another key for it, not title. I would also be wary of expecting to share the icon and logo due to their proportion requirements.If feed is already established as a site key, then stick with it, I guess, though personally I distinguish between the blog and the feed.